hoot-core-test/src/test/cpp/hoot/core/io/OsmApiWriterTestServer.cpp

```diff
@@ -31,6 +31,9 @@
 #include <hoot/core/io/OsmApiWriter.h>
 #include <hoot/core/util/Log.h>
 
+//  Standard
+#include <regex>
+
```

Comment:
I'm going to remove this right before merging.  It was a leftover from something I'd tried earlier and didn't remove.

Reply:
OK